runtime/moonbase/src/xcm_config.rs

```diff
@@ -265,9 +267,26 @@ impl frame_support::traits::Contains<RuntimeCall> for SafeCallFilter {
 }
 
 parameter_types! {
+	/// Location of Asset Hub
+	pub AssetHubLocation: Location = Location::new(1, [Parachain(1000)]);
```

Comment:
For moonbase, asset hub parachain id is `1001`
```suggestion
	pub AssetHubLocation: Location = Location::new(1, [Parachain(1001)]);
```